src/ankit/CodingExams/StringIdentical.java

```diff
@@ -0,0 +1,37 @@
+package ankit.CodingExams;
+
+import java.util.Scanner;
+
+public class StringIdentical {
+
+	void stringIdenticalLogic(String s1, String s2) {
+		boolean flag = false;
+		for (int index = 0; index < s1.length(); index++) {
+			char tempString1 = s1.charAt(index);
+			for (int innerIndex = 0; innerIndex < s2.length(); innerIndex++) {
+				char tempString2 = s2.charAt(innerIndex);
+				if (tempString1 == tempString2)
+					flag = true;
```

Comment:
you should write != and if this is the case then update flag value and break the loop